Runtime/Game/Resources/LootLockerConfig.cs

```diff
@@ -23,6 +23,7 @@ public static LootLockerConfig Get()
             if (settingsInstance != null)
             {
                 settingsInstance.ConstructUrls();
+                settingsInstance?.CheckForSettingOverrides();
```

Comment:
I can do both. I wanted it in the function so that if a function call is added from somewhere else that forgets a guard it wouldn't break anything (otherwise the code would be compiled and shipped with all builds).